src/main/java/tictactoe/UnbeatableComputerPlayer.java

```diff
@@ -0,0 +1,25 @@
+package tictactoe;
+
+public class UnbeatableComputerPlayer implements Player {
+    private final Display display;
+    private final String mark;
+    private final Minimax minimax;
+
```

Comment:
Nit pick: additional line... some of your future colleagues really care about this kind of thing - even if I'm not one of them :)

Reply:
Thanks for spotting this :)